### ISSUE TITLE
Everything functional after install except no story card UI in Firefox or Safari on OS X with Redmine 1.0.4

### ISSUE THREAD

**Issue by Halle** [SphereSoftware/easy_agile#7]
Hello,

I have an install which appears to be functionally working but it lacks any story card UI -- instead the stories are drawn as plain HTML.  Not an RMagick issue since the burndown draws perfectly, but I don't think it is a browser-related styling issue either since cards are unstyled in both browsers.

**Comment by Halle** [comment]
After looking into this a bit I have the impression that the HTML is fine but the plugin-specific .js and .css are not loaded on the page. Using the Safari web inspector I don't see any styles specific to the plugin being applied although the ea class names are visible in the source code for the page. This installation is the result of cloning the git repository into vendor/plugins so perhaps this is a side-effect. The js and css files are in the plugin assets folder.

**Comment by Halle** [comment]
OK, I was able to get this working after making changes to the source. I'm mystified at why my install is so different from what is expected after the install instructions, but I've uploaded my diffs to http://politepix.com/easy_agile_diffs.zip and I'll leave them there for a couple of weeks. The issues were that css wasn't loading due to caching being turned on, and several image urls would get 404s, and often the js files were only loaded at the end of the page after the content.  Now that I have it working, it's awesome!
